doc/developer_guide/conf.py

```diff
@@ -247,8 +247,8 @@
 #  documentclass [howto/manual]).
 latex_documents = [
     ('index', 'psyclone-dev.tex', u"PSyclone Developer Guide",
-     u'Rupert Ford, Joerg Henrichs, Iva Kavcic, Andrew Porter,\\\\ Sergi Siso'
-     ' and Lottie Turner', 'manual'),
+     u'Oakley Brunt, Rupert Ford, Joerg Henrichs, Iva Kavcic, \\\\'
```

Comment:
Thanks for adding your name :) There are a few more documentation options where it needs to be updated: `man_pages`, `texinfo_documents`, `epub_author` and `epub_publisher` (pretty much where you see Lottie's and my name.